src/components/utils/onfidoApi.js

```diff
@@ -24,8 +24,11 @@ const objectToFormData = (object) => {
 }
 
 const sendFile = (endpoint, data, token, onSuccess, onError) => {
-  data['sdk_source'] = 'onfido_web_sdk'
-  data['sdk_version'] = process.env.SDK_VERSION
+  data = {
+    sdk_source: 'onfido_web_sdk',
+    sdk_version: process.env.SDK_VERSION,
+    ...data
```

Comment:
MINOR: I would be put `...data` before, that way `sdk_source` and `sdk_version` cannot be overridden.